src/modules/io.wren

```diff
@@ -13,6 +13,7 @@ class FileSystem {
   foreign static save(path, buffer)
   foreign static prefPath(org, app)
   foreign static basePath()
+  foreign static createDirectory(path)
```

Comment:
This needs to be added to the documentation. The correct file for that is `docs/modules/io.md`.